modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -99,6 +99,8 @@ to the `libvirt` group:
 
```

Comment:
Suggest to add the following here instead: NOTE: You will need to log off and log back in to apply these changes.